src/lib/enum/promptElementType.ts

```diff
@@ -171,7 +171,7 @@ export const PROMPT_ELEMENT_TYPES = [
   new PromptElementType(
     'Git branch',
     // eslint-disable-next-line quotes
-    "$(git branch 2>/dev/null | grep ' ^* ' | colrm 1 2)",
+    "$(git branch 2>/dev/null | grep '*' | colrm 1 2)",
```

Comment:
Oh, apparently the whitespace was responsible for something being broken here. Is there any particular reason why you removed the caret (`^`) as well?

Reply:
Is the caret symbol really necessary? I never saw any git branch with asterisk in its name, so I assume there must always be only one asterisk in the output of `git branch`, that is alongside the current branch.
Please correct me if I'm wrong.

Reply:
No, it is not necessary. [Rule 5 on reference names](https://git-scm.com/docs/git-check-ref-format) forbids asterisks in branch names, so we can omit the caret.